From 95886fc4bade3b84739bc3ac7d5199ba6927ca96 Mon Sep 17 00:00:00 2001
From: Pan Xiuli <xiuli.pan@linux.intel.com>
Date: Wed, 21 Mar 2018 16:38:55 +0800
Subject: [PATCH 0878/1060] ASoC: SOF: add dsp_oops_offset for DSP panic

Store the offset to get the data stored in exception memory window.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>
---
 sound/soc/sof/sof-priv.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/sof/sof-priv.h b/sound/soc/sof/sof-priv.h
index 9d4ebe2ac767..e7e1b7b939a2 100644
--- a/sound/soc/sof/sof-priv.h
+++ b/sound/soc/sof/sof-priv.h
@@ -343,6 +343,7 @@ struct snd_sof_dev {
 	void __iomem *bar[SND_SOF_BARS];	/* DSP base address */
 	int mmio_bar;
 	int mailbox_bar;
+	size_t dsp_oops_offset;
 
 	/* debug */
 	struct dentry *debugfs_root;
-- 
2.17.1

